src/Moq/It.cs

```diff
@@ -276,6 +323,29 @@ public static TValue IsNotIn<TValue>(IEnumerable<TValue> items)
 			return Match<TValue>.Create(value => !items.Contains(value), () => It.IsNotIn(items));
 		}
 
+		/// <summary>
+		///   Matches any value that is not found in the sequence specified.
+		/// </summary>
+		/// <param name="items">The sequence of disallowed values.</param>
+		/// <param name="comparer">An <see cref="IEqualityComparer{T}"/> with which the values should be compared with.</param>
+		/// <typeparam name="TValue">Type of the argument to check.</typeparam>
+		/// <example>
+		///   The following example shows how to expect a method call with an integer argument
+		///   with value not found from a list.
+		///   <code>
+		///     var values = new List&lt;int&gt; { 1, 2, 3 };
+		///
+		///     mock.Setup(x => x.HasInventory(
+		///                         It.IsAny&lt;string&gt;(),
+		///                         It.IsNotIn(values)))
+		///         .Returns(false);
+		///   </code>
+		/// </example>
```

Comment:
As above, please either replace the example with a better fitting one, or remove this one altogether.